packages/design-tokens/jest.config.js

```diff
@@ -25,8 +28,4 @@ module.exports = {
       statements: 100,
     },
   },
-
-  // In jest.config.packages.js we are ignoring all index.ts files e.g. coveragePathIgnorePatterns: ['./src/index.ts'],
-  // We want to include index.ts in coverage so we override the coveragePathIgnorePatterns
-  coveragePathIgnorePatterns: [],
```

Comment:
This was for `design-system-tailwind-preset` updated now to ignore index files

### Before

![Screenshot 2024-11-18 at 2 33 41 PM](https://github.com/user-attachments/assets/994705f4-005f-4b79-a95d-b63d4c2f9081)


### After

![Screenshot 2024-11-18 at 2 34 07 PM](https://github.com/user-attachments/assets/d67ab120-b1a5-4732-86af-87a22bb8fd99)